be/src/main/java/codesquad/issuetracker/service/ImageService.java

```diff
@@ -46,4 +48,16 @@ public List<String> uploadFiles(List<MultipartFile> multipartFiles) {
 
         return fileUrls;
     }
+
+    private String createSavedFileName(String originalFilename) {
+        return UUID.randomUUID().toString().concat(getFileExtension(originalFilename));
```

Comment:
메소드 체이닝으로 한줄에 여러개의 마침표가 있으면 가독성이 떨어집니다.
. 단위로 줄을 분리 해주면 가독성이 좀 더 좋아집니다.
```suggestion
        return UUID.randomUUID()
        .toString()
        .concat(getFileExtension(originalFilename));
```